extensions/amp-form/0.1/amp-form.js

```diff
@@ -475,27 +475,24 @@ export class AmpForm {
       this.renderTemplate_(json || {});
       this.maybeHandleRedirect_(response);
     }, error => {
-      const message = 'Failed to parse response JSON:';
-      user().error(TAG, message, error);
-      rethrowAsync(message, error);
+      user().error(TAG, `Failed to parse response JSON: ${error}`);
```

Comment:
My theory is it was originally there to throw the error outside of the Promise chain so it would show up in the console. Since `user().error` performs that function too, it was equivalent.